client/app/widgets/ToggleLocale/__tests__/ToggleLocale.spec.jsx

```diff
@@ -17,23 +22,25 @@ describe('ToggleLocale', () => {
   });
 
   it('does nothing if the previous locale is the same as the selected', () => {
-    const wrapper = mount(component);
+    render(component);
     act(() => {
-      wrapper.find('select').prop('onChange')({
-        currentTarget: { value: 'en' },
+      fireEvent.select(screen.getByRole('combobox'), {
+        target: { value: 'en' },
       });
     });
     expect(axios.post).not.toHaveBeenCalled();
   });
 
-  it('sets the locale cookie and makes a post request if the selected locale is different from the previous', () => {
-    const wrapper = mount(component);
+  it('sets the locale cookie and makes a post request if the selected locale is different from the previous', async () => {
+    render(component);
     act(() => {
-      wrapper.find('select').prop('onChange')({
-        currentTarget: { value: 'es' },
+      fireEvent.change(screen.getByRole('combobox'), {
```

Comment:
Same as above here:

`act` can be removed and `userEvent` can be used instead of `fireEvent`.